app/Resources/public/less/index.less

```diff
@@ -216,6 +216,19 @@ html[lang|="en"] .promo-small-landscape.privacy {
     font-size: 1.3em;
 }
 
+.promo-large-portrait.ndg {
+    background-color: #27AE60;
+    background-image: url("/images/camera.svg");
```

Comment:
Can we somehow make text & icon have the same color (`fill: currentColor` or something along those lines)?